templates/android/docs/java/example.md.twig

```diff
@@ -55,8 +55,12 @@ Client client = Client(context)
                 return;
             }
 
+            {%~ if method.type != 'webAuth' %}
             {{ utils.resultTypeJava(sdk.namespace, method) }} response = ({{ utils.resultTypeJava(sdk.namespace, method) }}) o;
-        }
+            {%~ else %}
+            // Success
+            {%~ endif %}
+        }}
```

Comment:
Is this an extra double quote? This is the result for me:

```java
import io.appwrite.Client
import io.appwrite.services.Account

Client client = Client(context)
    .setEndpoint("https://[HOSTNAME_OR_IP]/v1") // Your API Endpoint
    .setProject("5df5acd0d48c2"); // Your project ID

Account account = new Account(client);

account.createOAuth2Session(
    "amazon",
    new Continuation<Void>() {
        @NotNull
        @Override
        public CoroutineContext getContext() {
            return EmptyCoroutineContext.INSTANCE;
        }

        @Override
        public void resumeWith(@NotNull Object o) {
            if (o instanceof Result.Failure) {
                Result.Failure failure = (Result.Failure) o;
                failure.exception.printStackTrace();
                return;
            }

            // Success
        }}
    }
);

```

Reply:
Yep it was, sorted now